themes/theme-gmd/extension-config.json

```diff
@@ -338,6 +338,17 @@
         "type": "json",
         "label": "Configure the display of product prices."
       }
+    },
+    "favorites": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": {
+        "limit": 100
```

Comment:
Having the same limit for frontend and backend is weird. Especially because this has to be set up for three different extensions that way: `Backend`, `theme-gmd` and `theme-ios11`

Reply:
Talked to PO. It's fine for now.